### PR TITLE
fix broken tests in windows

### PR DESCRIPTION
**Pull request by Illniyar**
when running npm test on windows, a "PATH" variable was added to the environment while in windows the path variable is "Path".
This caused two path variables in the env object in the tests, which caused the expected and actual results to diverge.

Since the use of `getPathVar` was removed from the lib in previous commits, I thought it ok to remove it's use in the test cases.

### CODE REVIEW

**Comment by kentcdodds** [comment]
Sorry, this wont work. We need to add the `Path` environment variable when spawning the script so we can have `./node_modules/.bin` in the path and anything else that `npm` puts in the path because the main use case is to use this in npm scripts. Merging this would cause a major regression across most users of `cross-env`

I would like to discuss the underlying problem you're describing though:

> This caused two path variables in the env object in the tests, which caused the expected and actual results to diverge.

I'm not sure why/how this could happen. Could you elaborate?

**Comment by Illniyar** [comment]
"We need to add the Path environment variable when spawning the script" -

but you don't, it was removed in https://github.com/kentcdodds/cross-env/commit/187c068f9bb12e010eb22f30ba0b8778e9edd6ca but the test code was not similarly updated.

In windows, what happens is that in the test the path variable is named "Path" but in the test cases you add another parameter named "PATH" (like in linux), because getPathVar() needs to be passed the platform value.
What this causes is that the spawn spy is called with the real process.env but we compare it to an object that has both a "Path" attribute and a "PATH" attribute.

Basically all tests fail on windows. Try to run the tests on a windows machine.

**Comment by kentcdodds** [comment]
Oh snap. My apologies! You're totally right!
